<?xml version="1.0" encoding="utf-8"?><testsuites name="pytest tests"><testsuite name="pytest" errors="10" failures="0" skipped="0" tests="10" time="7.996" timestamp="2025-08-12T05:50:35.819409+00:00" hostname="pkrvmsl9tci6h6u"><testcase classname="" name="tests.gui.test_enhanced_main_window" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/gui/test_enhanced_main_window.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/gui/test_enhanced_main_window.py:10: in &lt;module&gt;
    from PyQt5.QtWidgets import QApplication, QMessageBox, QFileDialog
E   ModuleNotFoundError: No module named 'PyQt5'</error></testcase><testcase classname="" name="tests.hardware.test_shimmer_integration" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/hardware/test_shimmer_integration.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/hardware/test_shimmer_integration.py:51: in &lt;module&gt;
    from tests.hardware.hardware_utils import (
tests/hardware/hardware_utils.py:20: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'

During handling of the above exception, another exception occurred:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/hardware/test_shimmer_integration.py:57: in &lt;module&gt;
    from .hardware_utils import (
tests/hardware/hardware_utils.py:20: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.hardware.test_thermal_camera" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/hardware/test_thermal_camera.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/hardware/test_thermal_camera.py:27: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.integration.virtual_environment.quick_test" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/integration/virtual_environment/quick_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.integration.virtual_environment.test_config" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/integration/virtual_environment/test_config.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.integration.virtual_environment.test_performance_benchmarks" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/integration/virtual_environment/test_performance_benchmarks.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.integration.virtual_environment.test_pytest_integration" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/integration/virtual_environment/test_pytest_integration.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.integration.virtual_environment.test_real_pc_integration" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/integration/virtual_environment/test_real_pc_integration.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.integration.virtual_environment.test_runner" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/integration/virtual_environment/test_runner.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase><testcase classname="" name="tests.test_virtual_environment_integration" time="0.000"><error message="collection failure">ImportError while importing test module '/home/runner/work/bucika_gsr/bucika_gsr/tests/test_virtual_environment_integration.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/test_virtual_environment_integration.py:15: in &lt;module&gt;
    from tests.integration.virtual_environment import (
tests/integration/virtual_environment/__init__.py:18: in &lt;module&gt;
    from .synthetic_data_generator import SyntheticDataGenerator
tests/integration/virtual_environment/synthetic_data_generator.py:17: in &lt;module&gt;
    import numpy as np
E   ModuleNotFoundError: No module named 'numpy'</error></testcase></testsuite></testsuites>